tests/unit/activation/test_nushell.py

```diff
@@ -13,9 +13,16 @@ def __init__(self, session):
 
             super().__init__(NushellActivator, session, cmd, "activate.nu", "nu")
 
+            self.activate_cmd = "overlay use"
```

Comment:
Shouldn't we make the activate command a list instead?

Reply:
I tried, but it would then try to run something like `'["overlay", "use"]' ...` literally spelling out the list as a string.

Reply:
I meant to rework the base class to allow the use of composed commands and change from single elements to list for the shells. Other shells can use a list of 1 length stuff.

Reply:
OK, as you want. You can do that before merging this PR and I can change it to be a list. This PR is meant for the next release of Nushell 27.9. but the old `source` method will still keep working after the release so there is no need to rush this PR.

Reply:
I don't have currently plan or bandwidth to do this myself, so I'm afraid as your feature needs this functionality you'll need to do it yourself here.

Reply:
Is it a problem to accept the current approach? It seems to be working correctly. The quoting change affects only the Nushell activator that affects only Nushell.

Reply:
While this approach works I'd prefer my proposal for ease of maintenance which is yes important for me.

Reply:
OK then, I'll see what I can do.